packages/comms/src/services/wsdl/WsDfu/v1.62/WsDfu.ts

```diff
@@ -0,0 +1,1455 @@
+import { IConnection, IOptions } from "../../../../connection";
+import { Service } from "../../../../espConnection";
+
+type base64Binary = number[];
```

Comment:
Did you double check that this is a `number[]` and not a `string` when its received?

Reply:
@jeclrsg - did you get anywhere with this?